packages/__examples__/.storybook/stories/generateComponentExamples.js

```diff
@@ -132,12 +116,14 @@ module.exports = function generateComponentExamples(
 
   const maybeAddExample = (props) => {
     const componentProps = getComponentProps(props)
-    const exampleProps = getExampleProps(props)
-    const key = uid()
-    const propsString = JSON.stringify(componentProps)
     const ignore = typeof filter === 'function' ? filter(componentProps) : false
-
-    if (!ignore && !PROPS_CACHE.includes(propsString)) {
+    if (ignore) {
+      return
+    }
+    const propsString = Object.entries(componentProps).sort().toString() //JSON.stringify(componentProps)
+    if (!PROPS_CACHE.includes(propsString)) {
+      const key = uid()
+      const exampleProps = getExampleProps(props)
       exampleCount++
```

Comment:
This method was the culprit for the bad performance. Its called >5000 times for some components and JSON.stringify is slow for such numbers. 
Now it exits early if something is filtered, and the Object.entries() method is way faster.